Sources/Sentry/SentryClient.m

```diff
@@ -634,6 +643,29 @@ - (void)recordLost:(BOOL)eventIsNotATransaction reason:(SentryDiscardReason)reas
     }
 }
 
+- (void)attachScreenshots:(SentryScope *)scope
+{
+#if SENTRY_HAS_UIKIT
+    if (!scope || !self.options.attachScreenshot)
+        return;
+
+    NSArray *shots = [SentryDependencyContainer.sharedInstance.screenshot appScreenshots];
```

Comment:
`l`: Naming this `shots` is quite confusing for me. I was thinking, who is shooting 😁?
```suggestion
    NSArray *screenshots = [SentryDependencyContainer.sharedInstance.screenshot appScreenshots];
```